store/mongo/datastore_mongo.go

```diff
@@ -496,9 +522,19 @@ func (db *DataStoreMongo) UpsertDeviceIntegrations(
 	updateOpts := mopts.FindOneAndUpdate().
 		SetUpsert(true).
 		SetReturnDocument(mopts.After)
-	collDevices := db.client.Database(*db.DbName).
-		Collection(CollNameDevices)
+	collDevices := db.Collection(CollNameDevices)
 	err := collDevices.FindOneAndUpdate(ctx, filter, update, updateOpts).
 		Decode(result)
 	return result, err
 }
+
+func (db *DataStoreMongo) GetAllDevices(ctx context.Context) (store.Iterator, error) {
+	collDevs := db.Collection(CollNameDevices)
+
+	return collDevs.Find(ctx,
+		bson.D{},
+		mopts.Find().
+			SetSort(bson.D{{Key: KeyTenantID, Value: 1}}),
+	)
+
```

Comment:
nit: do we need this extra empty line here?

Reply:
Nope, that's an unintended fmt :)